ams/api/src/main/java/com/netease/arctic/ams/api/metrics/MetricDefines.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netease.arctic.ams.api.metrics;
+
+import static com.netease.arctic.ams.api.metrics.MetricDefine.defineGauge;
+
+/** All metric defines. */
+public class MetricDefines {
+
+  // table optimizing status duration metrics
+  public static final MetricDefine TABLE_OPTIMIZING_STATE_IDLE_DURATION =
```

Comment:
Is it be better to define table related metrics as static attributes of TableMetrics. According to the previous review, metrics definitions should just be static where they are used, and table-related metrics should not be exposed to the api layer.

Reply:
fixed.